PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -3052,14 +3075,11 @@ int Progress::thread()
 
 int Progress::createProgressWindow()
 {
-	DWORD styleEx = WS_EX_OVERLAPPEDWINDOW;
-	if (_hOwner)
-		styleEx |= WS_EX_TOOLWINDOW;
-
-	_hwnd = ::CreateWindowEx(styleEx,
+	_hwnd = ::CreateWindowEx(
+		WS_EX_TOOLWINDOW | WS_EX_OVERLAPPEDWINDOW | WS_EX_TOPMOST,
 		cClassName, _header, WS_POPUP | WS_CAPTION,
 		CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT,
-		_hOwner, NULL, _hInst, (LPVOID)this);

```

Comment:
> I don't think the explicit cast here can do any harm anyway so I won't bother making new patch-set.

Fair enough.

The issue in the Windows vulnerability (doesn't apply here, but the idea is applicable) was that  someone was casting the address of a `PVOID` (i.e. a `&PVOID`/`PVOID*`/`void**`) to a plain-old `PVOID`/`void*`; the C-style cast just goes ahead, and [nods encouragingly](http://www.underhanded-c.org/_page_id_2.html).